kinesumer.go

```diff
@@ -54,19 +57,20 @@ var DefaultOptions = Options{
 	ShardAcquisitionTimeout: 90 * time.Second,
 }
 
-func NewDefault(stream string) (*Kinesumer, error) {
+func NewDefault(stream, duration string) (*Kinesumer, error) {
 	return New(
 		kinesis.New(session.New()),
 		nil,
 		nil,
 		nil,
 		stream,
 		nil,
+		duration,
 	)
 }
 
 func New(kinesis k.Kinesis, checkpointer k.Checkpointer, provisioner k.Provisioner,
-	randSource rand.Source, stream string, opt *Options) (*Kinesumer, error) {
+	randSource rand.Source, stream string, opt *Options, duration string) (*Kinesumer, error) {
```

Comment:
What do you think about making this a `time.Duration` instead of a `string`? It would make the interface a little easier for consumers, and reduce the chance that they format the duration wrong.